tests/common.go

```diff
@@ -4909,6 +4909,10 @@ func GetCloudDriveDeviceSpecs() ([]string, error) {
 func StartTorpedoTest(testName, testDescription string, tags map[string]string, testRepoID int) {
 	TestLogger = CreateLogger(fmt.Sprintf("%s.log", testName))
 	SetTorpedoFileOutput(log, TestLogger)
+	tags["apps"] = strings.Join(Inst().AppList, ",")
```

Comment:
What does provisioner means here? Spawn also pushes similar key.. cross check once it might get overwritten